packages/atlas/src/utils/size.ts

```diff
@@ -1,4 +1,4 @@
-const units = ['bytes', 'KB', 'MB', 'GB']
+const units = [' bytes', 'KB', 'MB', 'GB']
```

Comment:
Is this white space intentional?

Reply:
Yep, because when we format the size, `10KB` looks good, but `10bytes` is a bit weird